ui/app/AppLayouts/Profile/panels/CommunitiesListPanel.qml

```diff
@@ -30,17 +30,19 @@ StatusListView {
         statusListItemTitle.font.pixelSize: 17
         statusListItemTitle.font.bold: true
         subTitle: model.description
-        tertiaryTitle: qsTr("%n member(s)", "", model.members.count)
+        tertiaryTitle: qsTr("%n member(s)", "", (model.members_count || model.members.count))
         asset.name: model.image
         asset.isImage: asset.name.includes("data")
         asset.isLetterIdenticon: !model.image
         asset.bgColor: model.color || Theme.palette.primaryColor1
         asset.width: 40
         asset.height: 40
+        // TODO: this model is including ALL communities and using the joined flag to set visibility
+        // this shouldn't be done this way, instead the list should come already filtered from the backend
         visible: model.joined
         height: visible ? implicitHeight: 0
 
-        onClicked: setActiveCommunityClicked(model.id)
+        onClicked: setActiveCommunityClicked(model.model_id || model.id)
```

Comment:
Same here

Reply:
as discussed, this is because of the issue of using `id` in ListModel there seems to be no way around it. ultimately `model.id` will disapear and every model should use `.model_id` by convention.